### PR TITLE
Time period dropdown

### PR DESCRIPTION
**Pull request by Sajni94**
updated the dropdown so it feeds in week/month/year/lifetime into TimePeriod component and then also added cards for the statboard, not sure if we want to create one card style as a component and then use that?

### CODE REVIEW

**Comment by sazzamac** [comment]
Resolve conflicts

**Comment by sazzamac** [comment]
I'm not quite sure what's changed here - is the dropdown reactive now? The homepage data doesn't change for me whenever I change the dropdown option

**Comment by Sajni94** [comment]
@sazzamac oh yeah sorry I think I stupidly took out the console.log , I've basically got the dropdown to update the value that it sends up to the TimePeriod component where we'll put the logic to update the ecoScore. Let me quickly update it with the console.log back in.

**Comment by sazzamac** [comment]
It's ok, you're right to leave console.log out, we should only really use those for testing unless they display helpful messages to developers. I'm a bit confused though - what is the TimePeriod component?

**Review by sazzamac — APPROVED** [review]

File: src/screens/HomeScreen.js [hunk 1]
```diff
@@ -37,23 +49,38 @@ const HomeScreen = () => {
...
 
         {/* Placeholder for statboard section */}
-        <View style={styles.statBoard}>
-          {/* Need to make this dynamic based on time selection / whether trees or ocean has been good or bad */}
-          <Text style={styles.statBoardText}>Unfortunately this week you have destroyed 10 trees and polluted the ocean with plastic.</Text>
-          <Image source={require('../../assets/score_slider.png')} style={styles.scoreSlider}/>
-
+        <View style={{width: 300}}>
+          <Card style={styles.card}>
+            <View style={styles.statBoard}>
```
Comment (KejPat) [inline src/screens/HomeScreen.js:54]: Should the cards be in their own component which we pass props to for the details needed for that card? As we are repeating it a lot?

File: src/screens/HomeScreen.js [hunk 2]
```diff
@@ -37,23 +49,38 @@ const HomeScreen = () => {
         {ecoScore === "E" && <Image source={DyingEarth} style={styles.globeImage}/>}
 
         {/* Placeholder for statboard section */}
-        <View style={styles.statBoard}>
-          {/* Need to make this dynamic based on time selection / whether trees or ocean has been good or bad */}
-          <Text style={styles.statBoardText}>Unfortunately this week you have destroyed 10 trees and polluted the ocean with plastic.</Text>
-          <Image source={require('../../assets/score_slider.png')} style={styles.scoreSlider}/>
-
+        <View style={{width: 300}}>
```
Comment (KejPat) [inline src/screens/HomeScreen.js:52]: This should probably be the Timeperiod component which we pass the time to. In the Timeperiod component will be what you've put between this view, I think

**Review by KejPat — APPROVED** [review]

**Pull request merged**